src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -467,7 +467,11 @@ private function validateForm()
                 $fieldName = 'field' . $field['id'];
 
                 // skip
-                if ($field['type'] == 'submit' || $field['type'] == 'paragraph' || $field['type'] == 'heading') {
+                if ($field['type'] == 'submit' ||
```

Comment:
I think it is cleaner to have the || in front, that way you know at the start if it is an or or and and